baseclasses/BaseRegTest.py

```diff
@@ -225,7 +229,10 @@ def default(self, obj):
                 else:
                     obj = numpy.ascontiguousarray(obj)
                     assert obj.flags["C_CONTIGUOUS"]
-                return dict(__ndarray__=obj.tolist(), dtype=str(obj.dtype), shape=obj.shape)
+                if obj.size == 1:
+                    return numpy.asscalar(obj)
```

Comment:
This prints a deprecation warning. Looks like `obj.item()` is the preferred way since v1.16.

Reply:
Good catch, will fix